src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>IsRecurringDonations2Enabled__c</fullName>
```

Comment:
@sam-knox Theses two new settings fields aren't in the Quip doc, but they will also need review & help text. Thanks.

Reply:
@force2b - Thanks for flagging those. I'll take a look.

Reply:
@meena-sfdc - These settings aren't in our Quip doc, so I'll add my suggested changes here:

Change <label> on line 7 to: "Enhanced Recurring Donations".

Add this <inlineHelpText> and <description> "This option enables Enhanced Recurring Donations, and offers you the latest Recurring Donation features. But it also changes how you might have worked with Recurring Donations in the past. For complete details, see the NPSP product documentation."

@force2b 
For StageName_Installment__c, is it possible for this to be a picklist? If a user has to enter text themselves, do they enter the label or API name for the picklist value? What happens if they enter an invalid value (or no value at all)?

Reply:
@sam-knox Custom Settings do not support pick list field types, however we will eventually build a custom UI for this field in NPSP Settings that will allow the user to select a Stage from a list of all active Stages.

Reply:
Good idea, @force2b

Reply:
@force2b - ok, thanks Mike.

@meena-sfdc For the StageName_Installment__c field, add this help text and description:

"The Opportunity Stage that NPSP applies to each new installment Opportunity when you create a Recurring Donation."

Reply:
@meena-sfdc - Use this for the help text and description for the Enhanced Recurring Donations field:

"This option enables Enhanced Recurring Donations, and offers you the latest Recurring Donation features. &lt;b&gt;WARNING&lt;/b&gt;: When you enable this option, NPSP changes your existing Recurring Donation records and the Recurring Donations object to match the data model for Enhanced Recurring Donations. &lt;b&gt;This is an irreversible process&lt;/b&gt;. For complete details, see the NPSP product documentation."

This is an update from what I had posted earlier.

Reply:
Thanks for the updates, @sam-knox

Reply:
@force2b , since we are deciding on labels and descriptions, perhaps we can revisit API names?

- `IsRecurringDonations2Enabled__c` might be changed to `IsEnhancementEnabled__c` ?
- Remove `_` from `StageName_Installment__c`  and rename it to `InstallmentOppStageName__c`?

Reply:
@RadSFDeveloper - Yes, if we can align API names and labels, let's do it!

Reply:
@force2b , not sure if I can add bold <b> tags in the description of this field, I am getting errors when I deploy to scratch org.

Reply:
@force2b and @sam-knox , I made changes as per your suggestion for description and helptext and api names. However, the helptext for Enhanced Recurring Donations is exceeding the character limit. It does not say error, however, it is getting trimmed.

Reply:
@force2b - Is it possible to implement this using a custom label and keep the text? If not, I can try to shorten the text.

Reply:
I think this issue is settled now by updating the text in the UI and pulling it in, plus we've removed the html tags because they're not generally supported in InlineHelpText. The custom UI that will be built will not actually render the checkbox on the screen for enabling.

Reply:
@force2b - Ok, so I don't need to try to shorten the text to under 255?

Reply:
It should be good, thanks.
Also, I think that what's showing up in the "Conversation" tab in the PR is out of date. I'm going to resolve this conversation since it seems not match the actual file contents in the "Files Changed" tab.